src/main/java/cz/cvut/kbss/termit/rest/TermController.java

```diff
@@ -722,9 +733,21 @@ public List<AbstractChangeRecord> getHistory(@Parameter(description = ApiDoc.ID_
                                                  @PathVariable String localName,
                                                  @Parameter(description = ApiDoc.ID_STANDALONE_NAMESPACE_DESCRIPTION,
                                                             example = ApiDoc.ID_STANDALONE_NAMESPACE_EXAMPLE)
-                                                 @RequestParam(name = QueryParams.NAMESPACE) String namespace) {
+                                                 @RequestParam(name = QueryParams.NAMESPACE) String namespace,
+                                                 @Parameter(description = "Change type used for filtering.")
```

Comment:
In the method above you use constants for `Parameter` description values, but here there are literals. It seems to me the constants can be used here as well.